docs/user/local-transaction.md

```diff
@@ -0,0 +1,34 @@
+# Local transaction
+The broker semantics of handling local transaction is based on the AMQP version 0-9-1 specification. The broker allows 
+publish and acknowledge operations to be batched into atomic units of work until client confirm it succeeds or fails by 
+doing commit or rollback. The following high-level sequence diagram depicting how the main flow of transaction works.
+
+![Local Transaction Sequence](../images/local-transaction-sequence.png)
+
+## Client library support
```

Comment:
Shall we specify some example clients jars we can use? For example rabbitmq,  andes-client

Reply:
+1. I mentioned about WSO2 andes-client and RabbitMQ amqp-client in the paragraph.